Replace ActivityExtensions.SetStatus by Activity.SetStatus (#2358)
open-telemetry/opentelemetry-dotnet-contrib: src/OpenTelemetry.Instrumentation.AWSLambda/CHANGELOG.md

## Unreleased

> * Following tags will be not added to spans when the exception is recorder:

Reviewer: ```suggestion
* The following tags are no longer added to spans when an exception is recorded:
```

Author: Fixed in 214cad3aba3be7d9bfb5597e3a2df4a8aed63209 with all other entries
